common/browser.go

```diff
@@ -486,6 +486,16 @@ func (b *Browser) Close() {
 	b.conn.Close()
 }
 
+// CloseContext i a short-cut function to close the current browser's context.
+// If there is not an active browser context, it returns an error.
+func (b *Browser) CloseContext() error {
+	if b.context == nil {
+		return errors.New("can not close context as none is active in browser")
+	}
```

Comment:
An alternative would be to make this `noop` if the browser has no active browser context, but I think it's better to be "explicit" and give feedback to the user in this case.